test/unit/lodash.spec.js

```diff
@@ -0,0 +1,568 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var datasets = require('../../app/datasets');
+
+describe('Lodash training', function () {
+    var params;
+
+    describe('camelCase', function () {
+        before(function () {
+            params = datasets.camelCase();
+        });
+        it('should match types of passing elements', function () {
+            expect(params[0]).to.be.a('string');
+            expect(params).to.have.length(1);
+        });
+        it('should convert string to camel case', function () {
+                expect(_.camelCase.apply(_, params)).to.eql('someStringToTestFunction');
+            }
+        );
+    });
+
+    describe('capitalize', function () {
+        before(function () {
+            params = datasets.capitalize();
+        });
+        it('should match types of passing elements', function () {
+            expect(params[0]).to.be.a('string');
+            expect(params).to.have.length(1);
+        });
+        it('should convert the first character of string to upper case and the remaining to lower case', function () {
+                expect(_.capitalize.apply(_, params)).to.eql('Some string to test function');
+            }
+        );
+    });
+
+    describe('deburr', function () {
+        before(function () {
+            params = datasets.deburr();
+        });
+        it('should match types of passing elements', function () {
+            expect(params[0]).to.be.a('string');
+            expect(params).to.have.length(1);
+        });
+        it('should remove combining diacritical marks from string', function () {
```

Comment:
If this should remove extra marks, it should be some extra marks in text before...